src/scripts/wind_speed_retrieval.py

```diff
@@ -0,0 +1,182 @@
+import xsarsea
+from xsarsea import windspeed
+import xsar
+
+import xarray as xr
+import numpy as np
+
+import os,re
+import cv2
+
+from xsarsea.utils import _load_config
+config = _load_config()
+
+# output folder to store L2 products                 
+
+def make_L2(safe_path):
+    out_file = config['out_folder'] + safe_path.split("/")[-1] + "/output.nc"
+    if os.path.exists(out_file):
+        print("OK--", safe_path.split('/')[-1], "already treated")
```

Comment:
is it possible to go for `logging` rather than `print` here?

Reply:
it's done.